packages/forge/blocks/segment/actions/trackPage.ts

```diff
@@ -0,0 +1,78 @@
+import { createAction, option } from '@typebot.io/forge'
+import { Analytics } from '@segment/analytics-node'
+import { auth } from '../auth'
+
+export const trackPage = createAction({
+  auth,
+  name: 'Track Chatbot View',
```

Comment:
```suggestion
  name: 'Track',
```

Reply:
So, tracking page views vs tracking events are two different things in Segment. I can rename this to just 'Page' which makes it aligned with Segment, and then the Track Event can just be Event, does that work?

Reply:
Update name to Page, to align with Segment spec